chirp/drivers/ga510.py

```diff
@@ -514,7 +691,6 @@ def get_settings(self):
             'scanmode': ['TO', 'CO', 'SE'],
             'pttid': ['Off', 'BOT', 'EOT', 'Both'],
             'alarm_mode': ['Site', 'Tone', 'Code'],
-            'workmode': ['VFO', 'Chan'],
```

Comment:
Why is this removed? Shouldn't you just conditionally add to this list either just "workmode" or "workmodea" and "workmodeb"? It should reduce the amount of code you're adding below.